packages/python-sdk/e2b/session/filesystem.py

```diff
@@ -52,43 +52,52 @@ def __init__(self, session: SessionConnection):
     #         self.service_name, "writeBase64", [path, base64_content]
     #     )
 
-    async def read(self, path: str) -> str:
+    async def read(self, path: str, timeout: int = None) -> str:
         """
         Reads the whole content of a file as an array of bytes.
 
         :param path: path to a file
+        :param timeout: Timeout for the call
```

Comment:
Do we want all parameters descriptions to start with with capital letter or not?